yarn-project/noir-contracts/src/contracts/token_bridge_contract/src/main.nr

```diff
@@ -138,6 +138,13 @@ contract TokenBridge {
         1
     }
     /// docs:end:exit_to_l1_private
+
```

Comment:
Why is the `content` ordered differently than the function for public deposit? 

```
function = (to: AztecAddress, amount: Field, canceller: EthereumAddress)
content = (amount: Field, to: AztecAddress, canceller: EthereumAddress)
```

Reply:
The withdraw content hash computation is also reordering the arguments before hashing.